File: Python/1116.py
def countSecondInTime(h, m, s):
    return h * 3600 + m * 60 + s


h1, m1, s1 = map(int, input().split())
h2, m2, s2 = map(int, input().split())

totalSec1 = countSecondInTime(h1, m1, s1)
totalSec2 = countSecondInTime(h2, m2, s2)

print(abs(totalSec1 - totalSec2))

File: C++/1116.cpp
#include <iostream>
using namespace std;


int secondsInTime(int h, int m, int s)
{
    return h * 3600 + m * 60 + s;
}


int main()
{
    int h1, m1, s1, h2, m2, s2, totalSec1, totalSec2;

    cin >> h1 >> m1 >> s1;
    cin >> h2 >> m2 >> s2;

    totalSec1 = secondsInTime(h1, m1, s1);
    totalSec2 = secondsInTime(h2, m2, s2);

    cout << totalSec2 - totalSec1 << endl;

    return 0;
}
